book/graphics.md

```diff
@@ -212,15 +212,21 @@ every letter in the same place, so they all overlap! Let's fix that:
 
 ``` {.python expected=False}
 HSTEP, VSTEP = 13, 18
-x, y = HSTEP, VSTEP
+cx, cy = HSTEP, VSTEP
```

Comment:
What does cx stand for?

Reply:
They text tries to explain it as "cursor x" / "cursor y".

Reply:
cursor_x then?